pkg/internal/apis/acme/fuzzer/fuzzer.go

```diff
@@ -42,10 +42,9 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 				s.Spec.IssuerRef.Kind = v1alpha2.IssuerKind
 			}
 		},
-		func(s *acme.ACMEIssuerDNS01ProviderWebhook, c fuzz.Continue) {
-			c.FuzzNoCustom(s) // fuzz self without calling this function again
+		func(s *apiext.JSON, c fuzz.Continue) {
 			// ensure the webhook's config is valid JSON
-			s.Config = &apiext.JSON{Raw: []byte("{}")}
+			s.Raw = []byte("{}")
```

Comment:
Maybe add a comment here to say why this is now `.Raw` to reduce bus factor changing this code again in future :sweat_smile:

Reply:
This was always `.Raw` (see the line it replaced) - the difference is that we're defaulting the actual structure that needs defaulting instead of defaulting a structure containing the structure that needs defaulting. Not sure if it's really worth adding a comment for that...?